test/unit/standalone_spec.js

```diff
@@ -54,11 +54,11 @@ describe('Typeahead', function() {
           input: {
             $input: {}
           },
-          open: sinon.spy(),
-          close: sinon.spy(),
-          getVal: sinon.spy(),
-          setVal: sinon.spy(),
-          destroy: sinon.spy()
+          open: sinon.stub().returns('hello'),
+          close: sinon.stub().returns('hello'),
+          getVal: sinon.stub().returns('hello'),
```

Comment:
I think that the underlying Typeahead has a problem, that is why I replaced the call with getInputValue instead of the getQuery. Here we stub the the Typeahead but I don't think it solves the issue.